datasets/me/acp_peps/crawler.py

```diff
@@ -0,0 +1,212 @@
+
+        # Emit a relative
+        relative = context.make("Person")
+        relative.id = context.make_id(relative_name, relative_surname, maiden_name)
+        relative.add("address", address)
+        relative.add("nationality", nationality)
+        h.apply_name(
+            relative,
+            first_name=relative_name,
+            last_name=relative_surname,
+            maiden_name=maiden_name,
+        )
+        address_ent = h.make_address(
+            context,
+            full=address,
+            city=city,
+            lang="cnr",
+        )
+        h.copy_address(relative, address_ent)
+        context.emit(relative)
+
+        # Emit a relationship
+        rel = context.make("Family")
+        rel.id = context.make_id(person_entity.id, relationship, relative.id)
+        rel.add("relationship", relationship)
+        rel.add("person", person_entity.id)
+        rel.add("relative", relative.id)
+
+        context.emit(rel)
+        context.audit_data(row)
+
+
+def make_affiliation_entities(
+    context: Context, entity: Entity, function, row: dict, filing_date, report_id
+) -> List[Entity]:
+    """Creates Position and Occupancy provided that the Occupancy meets OpenSanctions criteria.
+    * A position's name include the title and the name of the legal entity
+    * All positions (and Occupancies, Persons) are assumed to be Montenegrin
+    """
+
+    organization = row.pop("ORGANIZACIJA")
+    start_date = row.pop("DATUM_POCETKA_OBAVLJANJA", None)
+    end_date = row.pop("DATUM_PRESTANKA_OBAVLJNJA")
+    context.audit_data(
+        row, ignore=["ORGANIZACIJA_IMENOVANJA", "ORGANIZACIJA_SAGLASNOSTI"]
+    )
+
+    position_name = f"{function}, {organization}"
+    entity.add("position", position_name)
+
+    position = h.make_position(
+        context,
+        position_name,
+        # topics=["gov.national"],  # for testing
+        country="ME",
+    )
+    apply_translit_full_name(
+        context, position, "cnr", position_name, TRANSLIT_OUTPUT, POSITION_PROMPT
+    )
+
+    categorisation = categorise(context, position, is_pep=True)
+    # Check categorisation topics match and fetch relatives
+    if "gov.national" in categorisation.topics:
+        relatives = fetch_csv(context, report_id, "csv_clanovi_porodice_")
+        crawl_relative(context, entity, relatives)
+
+    occupancy = h.make_occupancy(
+        context,
+        entity,
+        position,
+        no_end_implies_current=True,
+        categorisation=categorisation,
+        propagate_country=True,
+        start_date=start_date,
+        end_date=end_date,
+    )
+    entities = []
+    if occupancy:
+        # Switch to declarationDate, once it's introduced in FtM
+        occupancy.add("date", filing_date)
+        entities.extend([position, occupancy])
+    return entities
+
+
+def crawl_person(context: Context, person):
+    full_name = person.pop("imeIPrezime")
+    # position = person.pop("nazivFunkcije")
```

Comment:
reminder to use this as a fallback if there aren't downloadable declarations. But lets get the declaration crawling dialed first.